src/components/Cards/CreateAccount.vue

```diff
@@ -65,12 +71,11 @@
                     @click.stop
                     v-on="on"
                   >
-                    Terms of Use
+                    Terms of Use.
                   </a>
                 </template>
                 Opens in new window
```

Comment:
`Open link in new tab` (like Chrome) sounds better technically

Reply:
same here, we should ask UX